src/molecules/dialog/index.tsx

```diff
@@ -32,33 +33,34 @@ export default function AutoformDialog({
   triggerData,
 }: AutoformDialogProps) {
   const [values, setValues] = useState<any>(triggerData || {});
+
   return (
     <Dialog open={open} onOpenChange={onOpenChange}>
-      <DialogContent className="sm:max-w-[425px]">
-        <DialogHeader>
-          <DialogTitle>{action?.cta}</DialogTitle>
-          <DialogDescription>{action?.description}</DialogDescription>
-        </DialogHeader>
-        <div className="grid gap-4 py-4">
-          <AutoForm
-            {...action?.autoFormArgs}
-            onParsedValuesChange={(e) => {
-              setValues(e);
-            }}
-            values={values}
-            onSubmit={(formData) => {
-              action?.callback(formData, triggerData);
-            }}
-          >
-            {action?.autoFormArgs?.children}
-            <AutoFormSubmit className="float-right">
-              Save Changes
-            </AutoFormSubmit>
-          </AutoForm>
-        </div>
-        <DialogFooter>
-          {/* TODO: Dialog footer to add whatever children we need */}
-        </DialogFooter>
+      <DialogContent className="sm:max-w-[425px] px-0">
```

Comment:
Would it be enough to have this PR with the next? 
```suggestion
      <DialogContent className="sm:max-w-[425px]  max-h-[70vh] overflow-auto">
```